CMakeLists.txt

```diff
@@ -986,7 +986,14 @@ set(MIXXX_INSTALL_BINDIR ".")
 set(MIXXX_INSTALL_DATADIR ".")
 set(MIXXX_INSTALL_DOCDIR "./doc")
 set(MIXXX_INSTALL_LICENSEDIR "./doc")
-if (UNIX)
+if (APPLE)
+  set(MIXXX_INSTALL_BINDIR "${CMAKE_INSTALL_BINDIR}")
+  set(MACOS_BUNDLE_NAME Mixxx)
+  set(MIXXX_INSTALL_PREFIX "${MACOS_BUNDLE_NAME}.app/Contents/Resources")
+  set(MIXXX_INSTALL_DATADIR "${MACOS_BUNDLE_NAME}.app/Contents/Resources")
+  set(MIXXX_INSTALL_DOCDIR "${MIXXX_INSTALL_DATADIR}")
+  set(MIXXX_INSTALL_LICENSEDIR "${MIXXX_INSTALL_DATADIR}/licenses/mixxx")
```

Comment:
Is the subdirectory `/mixxx` intended? This should be redundant, because the application name should already be included in MACOS_BUNDLE_NAME in one of the parent directories. No need for redundant scoping.

Reply:
good catch